api/src/config.ts

```diff
@@ -51,7 +51,7 @@ export const config = {
       host: env.TR_DB_HOST || '127.0.0.1',
       port: parseInt(env.TR_DB_PORT, 10) || 3306,
       username: env.TR_DB_USER || 'root',
-      password: env.TR_DB_PASSWORD || 'root',
```

Comment:
@bvenceslas I think that will crush some things... such default password used only during development to be able to connect to local DB with some default password I think....

Reply:
i've set it to default